packages/block-library/src/image/view.js

```diff
@@ -24,11 +30,46 @@ let isTouching = false;
  */
 let lastTouchTime = 0;
 
+/**
+ * Returns the appropriate src URL for an image
+ *
+ * @param {Object} imageMetadata Image metadata object
+ * @return {string} The source URL
+ */
+function getImageSrc( imageMetadata ) {
+	return (
+		imageMetadata.uploadedSrc ||
+		'data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs='
+	);
+}
+
+/**
+ * Returns the appropriate srcset for an image
+ *
+ * @param {Object} imageMetadata Image metadata object
+ * @return {string} The srcset value
+ */
+function getImageSrcset( imageMetadata ) {
+	return imageMetadata.lightboxSrcset || '';
+}
+
+/**
+ * Returns the appropriate sizes attribute for an image
+ *
+ * @param {Object} imageMetadata Image metadata object
+ * @return {string} The sizes value, defaulting to 100vw
+ */
+function getImageSizes( imageMetadata ) {
+	return imageMetadata.lightboxSizes || '100vw';
+}
```

Comment:
Minor improvement, to add typing for the passed prop:

```suggestion
/**
 * Returns the appropriate src URL for an image.
 *
 * @param {string} uploadedSrc - Full size image src.
 * @return {string} The source URL.
 */
function getImageSrc( { uploadedSrc } ) {
	return (
		uploadedSrc ||
		'data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs='
	);
}

/**
 * Returns the appropriate srcset for an image.
 *
 * @param {string} lightboxSrcset - Image srcset.
 * @return {string} The srcset value.
 */
function getImageSrcset( { lightboxSrcset } ) {
	return lightboxSrcset || '';
}

/**
 * Returns the appropriate sizes attribute for an image.
 *
 * @param {string} lightboxSizes - Image responsive sizes attribute.
 * @return {string} The sizes value, defaulting to 100vw.
 */
function getImageSizes( { lightboxSizes } ) {
	return lightboxSizes || '100vw';
}
```

Reply:
Made the changes ✅